src/List.test.jsx

```diff
@@ -37,9 +38,14 @@ describe('List', () => {
     });
 
     it('show delete button', () => {
-      const { container } = listRender(tasks);
+      const { container, getAllByText } = listRender(tasks);
 
       expect(container).toHaveTextContent('완료');
+
+      expect(handleClickDelete).not.toBeCalled();
+      const deleteButtons = getAllByText('완료');
+      deleteButtons.forEach((deleteButton) => fireEvent.click(deleteButton));
```

Comment:
```suggestion
      deleteButtons.forEach((deleteButton) =>
        fireEvent.click(deleteButton)
      );
```

그리고 45-48은 맥락이 45, 46, 47, 48 라인 모두 달라보이네요.

한 줄씩 띄워주면 좋겠습니다.

이 부분 테스트 잘하셨어요.

적절히 줄바꿈 해주는 게 가독성에 좋습니다.